backend/flask_api.py

```diff
@@ -201,5 +201,112 @@ def search():
         return (str(e), 400)
 
 
+@app.route('/api/projects/search/simple/<int:offset>/<int:count>/', methods=['GET'])
+def search_simple(offset, count):
+    """Search projects
+
+    Returns:
+        res (json): JSON containing Projects and metadata
+
+    """
+    text = request.args.get("q", type=str)
+    sorting = request.args.get('sort', type=str)
+    order = request.args.get('order', type=str)
+    # ^3 is boosting the attribute, *_is allowing wildcards to be used
+    request_string = '{"from" : 1, "size" : 1, "query": {"multi_match" :' \
```

Comment:
Could you create this with a real dictionary instead of loading from a string, which will happen every time at runtime and can probably not be optimized by the compiler at all and is also error prone, if you want to make adjustments later.